src/app/containers/Include/amp/VjAmp.jsx

```diff
@@ -7,6 +7,39 @@ const IncludeGrid = styled(GridItemConstrainedMedium)`
   display: grid;
 `;
 
+const PocketTapArea = styled.div`
+  &&& {
+    display: block;
+    height: 70px;
```

Comment:
I'm not entirely sure if it's written anywhere but we should always use REM in our codebase so the height responds to the browser font size choice for accessibility reasons.